### ISSUE TITLE
⌨ `ExternalLink` with secondary colour has insufficient colour contrast

### ISSUE THREAD

**Issue by esizer** [GCTC-NTGC/gc-digital-talent#6766]
## ⌨ Accessibility Issue

On our register page we have a "Login instead" link using the `ExternalLink` component with the `secondary` colour which is not meeting the min. colour contrast ratio of 4.5:1.

### WCAG 2.1 Criteria

[Elements must have sufficient color contrast](https://dequeuniversity.com/rules/axe/4.7/color-contrast?application=axeAPI)

## 🦋 Expected Behaviour

`ExternalLink` component has contrast ratio >= 4.5:1.

## 📋 Steps to Reproduce

1. Navcigate to `/register`
3. Confirm the "Log in instead" link is too light on the white background

## 📸 Screenshot

![Screenshot 2023-05-31 095135](https://github.com/GCTC-NTGC/gc-digital-talent/assets/4127998/0572c44d-9a77-423a-9b25-e833d850877f)

## 🙋‍♀️ Proposed Solution

- [ ] Use darker shade of `secondary` colour for `ExternaLink`

## ✅ Acceptance Criteria

A set of assumptions which, when tested, verify that the issue was addressed.

- [ ] Secondary `ExternalLink` component meets min contrast of 4.5:1

**Comment by tristan-orourke** [comment]
Have a working example on the Login page
